collector/settings.mdx

```diff
@@ -6,17 +6,22 @@ backlink_title: 'pganalyze Collector'
 
 import ToC from '../components/Toc'
 
-The collector can be configured either through an `INI` config file (in a package-based install) or environment
-variables (typically for running via its Docker image). Most settings can be configured through either mechanism.
-If both are present, the config file takes precedence. Note that a single collector instance can monitor more than
-one database server, though this is not supported when configuring through environment variables.
+The collector can be configured through one of three mechanisms:
 
-For the `INI`-based setup, the `[pganalyze]` section describes settings that apply to all servers. Other sections
+1. Package-based install: `INI` config file, commonly in `/etc/pganalyze-collector.conf`
+2. Docker-based install: `INI` config passed via the `CONFIG_CONTENTS` environment variable
+3. Docker-based install: Individual environment variables for each setting (e.g. `DB_HOST`)
+
+Most settings can be configured through either `INI` config settings or individual environment variables. When you set individual environment variables in addition to the `INI` config, the environment variable settings apply to all monitored
```

Comment:
This is missing a line break (only mentioning b/c the rest of the document is pretty consistent)